sphinx/source/docs/dev_guide.rst

```diff
@@ -36,29 +36,42 @@ you must first be able to build BokehJS.
 Building BokehJS
 ----------------
 
-Install Prerequisites
-~~~~~~~~~~~~~~~~~~~~~
-
 The BokehJS build process is handled by `Grunt <http://gruntjs.com/>`_,
 which in turn depends on `Node.js <http://nodejs.org/>`_.
 Grunt is used to compile CoffeeScript, Less and Eco sources, combine JavaScript resources,
 and generate optimized and minified ``bokeh.js`` and ``bokeh.css`` files.
 
-You can download and install Node.js directly, or use
-`conda <http://conda.pydata.org/>`_ to install Node.js
+
+Install npm and node
+~~~~~~~~~~~~~~~~~~~~
+
+First, install Node.js and npm (node package manager).
+You can download and install these directly, or use
+`conda <http://conda.pydata.org/>`_ to install them
 from the Bokeh channel on `Binstar <https://binstar.org>`_::
 
     $ conda install -c bokeh nodejs
+    
+Alternatively, on Ubuntu you can use ``apt-get``::
+    
+    $ apt-get install npm node
+
+
+Install Grunt and necessary plugins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install grunt itself::
 
-.. note:: The following commands should be executed in the ``bokehjs``
-          subdirectory of the Bokeh source checkout.
+    $ npm install -g grunt-cli
```

Comment:
There's really no reason for anyone to ever do `node_modules/.bin/grunt` either, really. The `setup.py` handles everything. It's just the way the doc is currently written talks about "testing things out" first, and says to use `grunt` for that, but we could just leave that out. The only people I ever imagine would need to run `grunt` by hand would be someone doing pure JS dev on BokehJS. Could add a note mentioning the global install "may be useful if you are doing pure JS dev".